fatal error: rocm/include/rocblas.h: No such file or directory

<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Build/Install

### Have you reproduced the bug with TF nightly?

Yes

### Source

source

### Tensorflow Version

nightly

### Custom Code

Yes

### OS Platform and Distribution

Linux Manjaro

### Mobile device

_No response_

### Python version

3.10

### Bazel version

5.3.0

### GCC/Compiler version

12.2.0

### CUDA/cuDNN version

_No response_

### GPU model and memory

AMD Vega 10

### Current Behaviour?

```shell
While compiling from sources I have the error

´fatal error: rocm/include/rocblas.h: No such file or directory´

I can fix that replacing

´#include "rocm/include/rocblas.h"´

by

´#include "rocm/include/rocblas/rocblas.h"´

at tensorflow/compiler/xla/stream_executor/rocm/rocblas_wrapper.h
```


### Standalone code to reproduce the issue

```shell
./configure
bazel build --config=rocm //tensorflow/tools/pip_package:build_pip_package
```


### Relevant log output

```shell
ERROR: /home/pepe/CoreMarine/pinn/tensorflow/tensorflow/compiler/xla/stream_executor/rocm/BUILD:171:11: Compiling tensorflow/compiler/xla/stream_executor/rocm/rocm_blas.cc failed: (Exit 1): crosstool_wrapper_driver_is_not_gcc failed: error executing command external/local_config_rocm/crosstool/clang/bin/crosstool_wrapper_driver_is_not_gcc -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer ... (remaining 128 arguments skipped)
In file included from tensorflow/compiler/xla/stream_executor/rocm/rocm_blas.cc:18:
./tensorflow/compiler/xla/stream_executor/rocm/rocblas_wrapper.h:23:10: fatal error: rocm/include/rocblas.h: No such file or directory
   23 | #include "rocm/include/rocblas.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
Target //tensorflow/tools/pip_package:build_pip_package failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 40.598s, Critical Path: 30.80s
INFO: 810 processes: 370 internal, 440 local.
FAILED: Build did NOT complete successfully
```
</details>

--- a/tensorflow/compiler/xla/stream_executor/rocm/rocblas_wrapper.h
+++ b/tensorflow/compiler/xla/stream_executor/rocm/rocblas_wrapper.h
@@ -20,7 +20,7 @@ limitations under the License.
 #ifndef TENSORFLOW_COMPILER_XLA_STREAM_EXECUTOR_ROCM_ROCBLAS_WRAPPER_H_
 #define TENSORFLOW_COMPILER_XLA_STREAM_EXECUTOR_ROCM_ROCBLAS_WRAPPER_H_
 
-#include "rocm/include/rocblas.h"
+#include "rocm/include/rocblas/rocblas.h"
 #include "tensorflow/compiler/xla/stream_executor/gpu/gpu_activation.h"
 #include "tensorflow/compiler/xla/stream_executor/lib/env.h"
 #include "tensorflow/compiler/xla/stream_executor/platform/dso_loader.h"

